### ISSUE TITLE
Environment error, incompatible CPU detected. Hypervisor fails

### ISSUE THREAD

**Issue by saporator** [docker/for-mac#6745]
Hi all,

  - [X] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [X] I have uploaded Diagnostics
  - Diagnostics ID: 410F7287-DF06-4D9B-BACE-B869CFDF6626/20230227111730

### Expected behavior
I installed docker via "brew install docker" but as afterwards no desktop app was installed I installed the app from docker.com for Intel chip.

When I open the desktop app an error message is shown:
![screenshot](https://user-images.githubusercontent.com/59700253/221553586-08130814-2327-438d-9ddd-db3ee8f936d2.png)

### Actual behavior
App is closed with error message

### Information
  - macOS Version: 12.6.3
  - Intel chip or Apple chip: 2,66 GHz Quad-Core Intel Xeon
  - Docker Desktop Version: latest from web page. In terminal "Docker version 20.10.22, build 3a2c30b" is shown.

### Output of `/Applications/Docker.app/Contents/MacOS/com.docker.diagnose check`
The diagnose check doesn't end, I needed to terminate it.
[check.txt](https://github.com/docker/for-mac/files/10839296/check.txt)

### Steps to reproduce the behavior
Open app

**Comment by sam-thibault** [comment]
Here is a similar error reported in the docker forums (with PRAM reset solution): https://forums.docker.com/t/docker-desktop-3-5-2-wont-start-after-update-fail-hypervisor-framework-fails/112703

Can you share more details about the specific Mac model?

**Comment by saporator** [comment]
I have a MacPro (Early 2009), 24 GB RAM, which I patched with OpenCore to be able o run e.g. Monterey.

I found on another page that the issue is Hypervisor.framework is not supported by my CPU:
sysctl kern.hv_support
kern.hv_support: 0

https://forums.docker.com/t/fatal-error-incompatible-cpu-detected/9162/5 

In principle I might not need Docker App, I only wanted to install paperless-ngx, but wihout Docker I struggle so far to get it installed.
